ting_subsearch_common.module

```diff
@@ -296,3 +296,26 @@ function ting_subsearch_common_libraries_info() {
     ],
   ];
 }
+
+/**
+ * Transform strings to floats in variables.
+ *
+ * @param $num
+ *
+ * @return float
+ */
+function ting_subsearch_common_tofloat($num) {
+  $dotPos = strrpos($num, '.');
+  $commaPos = strrpos($num, ',');
+  $sep = (($dotPos > $commaPos) && $dotPos) ? $dotPos :
+    ((($commaPos > $dotPos) && $commaPos) ? $commaPos : false);
+
+  if (!$sep) {
+    return floatval(preg_replace("/[^0-9]/", "", $num));
+  }
+
+  return floatval(
+    preg_replace("/[^0-9]/", "", substr($num, 0, $sep)) . '.' .
+    preg_replace("/[^0-9]/", "", substr($num, $sep+1, strlen($num)))
+  );
+}
```

Comment:
A'int this an overkill? Truth is, I don't know what is the input.
What about - `(float) str_replace(['.',','], ['', '.'], $num) * 1.0` ?
```
php > $num = "1,2345";
php > var_dump((float) str_replace(['.',','], ['', '.'], $num) * 1.0);
php shell code:1:
double(1.2345)
php > $num = "1,2345.67";
php > var_dump((float) str_replace(['.',','], ['', '.'], $num) * 1.0);
php shell code:1:
double(1.234567)
php > $num = "1.2345,67";
php > var_dump((float) str_replace(['.',','], ['', '.'], $num) * 1.0);
php shell code:1:
double(12345.67)
```